src/schema/v2/artwork/artworkContextGrids/__tests__/defaultContext.test.ts

```diff
@@ -158,10 +159,7 @@ describe("Default Context", () => {
     expect(artistTitle).toEqual("Other works by Andy Warhol")
     expect(artistCtaTitle).toEqual("View all works by Andy Warhol")
     expect(artistctaHref).toEqual("/artist/andy-warhol")
-    expect(artistArtworks.edges.map(({ node }) => node.slug)).toEqual([
-      "artwork1",
-      "artwork2",
-    ])
+    expect(artistArtworks.edges.map(({ node }) => node.slug)).toEqual([])
```

Comment:
Maybe there are ideas/tips on how to make it so that when [the filter is active](https://github.com/artsy/metaphysics/blob/921ba0e68528cf2828567f10a2e10e2be0421200/src/schema/v1/artwork/artworkContextGrids/ArtistArtworkGrid.ts#L49), the elements are returned?